test/tests/restart/restart_transient_from_steady/steady_diffusion.i

```diff
@@ -53,30 +53,9 @@
 [Postprocessors]
   [pwr_avg]
     type = ElementAverageValue
-    block = '0'
     variable = power_density
     execute_on = 'initial timestep_end'
   []
-  [temp_avg]
-    type = ElementAverageValue
-    variable = Tf
-    block = '0'
-    execute_on = 'initial timestep_end'
-  []
-  [temp_max]
-    type = ElementExtremeValue
-    value_type = max
-    variable = Tf
-    block = '0'
-    execute_on = 'initial timestep_end'
-  []
-  [temp_min]
-    type = ElementExtremeValue
-    value_type = min
-    variable = Tf
-    block = '0'
-    execute_on = 'initial timestep_end'
-  []
```

Comment:
Why did you remove those? It looks like there is still a problem with their values

Reply:
No quite useful unless we are going to change the design

Reply:
BTW, these info can be printed out in the sub apps since `Tf` is calculated in the sub apps any way.

Reply:
I agree that it can be done in the subapp but it is still useful to test the transfer

Reply:
I add it back using `excute_on = final`.  You are able to see it only at the end of simulation.

Reply:
In that case, it makes sense!